chef/cookbooks/provisioner/templates/default/crowbar_join.suse.sh.erb

```diff
@@ -456,6 +456,15 @@ if [ "$MODE" == "setup" -o "$MODE" == "start" ]; then
     # Get our hostname
     HOSTNAME=$(hostname -f)
 
+    # stop ntpd before we run ntpdate, and start it again afterwards.
+    ntp=ntp
+    service ntp status > /dev/null || ntp=ntpd
```

Comment:
Sure, removing this variable, and then? Then it's not working for SLE 12, awesome!